internal/npm_install/generate_build_file.ts

```diff
@@ -206,7 +214,7 @@ function flattenDependencies(pkgs: Dep[]) {
 async function generateRootBuildFile(pkgs: Dep[]) {
   let buildFile = config.exports_directories_only ?
       printRootExportsDirectories(pkgs) :
-      printRoot(pkgs);
+      printRootLegacy(pkgs);
```

Comment:
maybe "ExportsAllFiles" is more descriptive than "Legacy"?

Reply:
either or